### ISSUE TITLE
formbuilder API headless

### ISSUE THREAD

**Issue by onno-timmerman** [wagtail/wagtail#6950]
### Is your proposal related to a problem?
Formbuilder has no API post method rendering the feature in headless useless. If I make a contact page and want to add a formbuilder then I would expect to have a post method to that page in the api.

### Describe the solution you'd like
Its trivial to add the formbuilder fields to the API but I can't find any way to post to the page. I would propose there is some documentation how to do it or some way to add a POST to the restfull api.

_Maybe something APIFields you can add APIPost(related_name)_ 

### Describe alternatives you've considered
I only see option to make custom model and serializer with view.

### Additional context
_If you are a Vue/Nuxt user check out this:_
https://vueformulate.com/guide/forms/generating-forms/

_You can simple map Formbuilder fields to this vue-formulate schema and you'll have your form with frontend validation and backend validation._

**Comment by prasen-ftech** [comment]
@onno-timmerman did you find any work around for this? I am also looking for API headless for wagtail formbuilder

**Comment by onno-timmerman** [comment]
No, the pressure wasn't high enough. A custom form was ok for then. But still miss a simple way to do it with a formbuilder.

**Comment by ammein** [comment]
I needed this too. I was looking to bind GraphQL for fetch only data by I also need a contact form POST API to post only the contact forms that has email capability too. I don't need POST API to edit pages,delete pages or any of that. Just a custom POST for Forms. To me, this is truly crucial feature for Headless CMS. I love speed development of Wagtail, but the form POST API is a bummer for me.
